addons/website/views/snippets/s_alert.xml

```diff
@@ -11,20 +11,30 @@
 </template>
 
 <template id="s_alert_options" inherit_id="website.snippet_options">
-    <xpath expr="." position="inside">
-        <div data-selector=".s_alert">
-            <we-colorpicker string="Color"
-                data-select-style="true"
-                data-css-property="background-color"
-                data-color-prefix="alert-"/>
+    <xpath expr="//div[@id='so_width']" position="before">
+        <div data-selector=".s_alert" data-js="Alert">
+            <we-select string="Type" data-apply-to=".fa.s_alert_icon">
+                <we-button data-select-class="fa-user-circle" data-trigger="s_alert_colorpicker" data-trigger-value="primary">Primary</we-button>
+                <we-button data-select-class="fa-user-circle-o" data-trigger="s_alert_colorpicker" data-trigger-value="secondary">Secondary</we-button>
+                <we-button data-select-class="fa-info-circle" data-trigger="s_alert_colorpicker" data-trigger-value="info">Info</we-button>
+                <we-button data-select-class="fa-check-circle" data-trigger="s_alert_colorpicker" data-trigger-value="success">Success</we-button>
+                <we-button data-select-class="fa-exclamation-triangle" data-trigger="s_alert_colorpicker" data-trigger-value="warning">Warning</we-button>
+                <we-button data-select-class="fa-exclamation-circle" data-trigger="s_alert_colorpicker" data-trigger-value="danger">Danger</we-button>
+            </we-select>
         </div>
-
+    </xpath>
+    <!-- Keep those options in separate xpath for options order -->
+    <xpath expr="//div[@id='so_width']" position="after">
```

Comment:
I don't see this `so_width`, you probably added it in the wrong commit... or maybe it is in a commit which is before in the tree but not in the PR order? Could you also name it differently? It will be confusing with the regular/small/full option which is also named width.

Reply:
Yes added in a separate commit, where `so_text_align` is also added on Alignment option which is adding text-center class etc.

Hm, this one is also named `Width`, thus the `so_width`.

Imo `so_width` is fine for this one, which actually play with the width (w-25 w-50..), the ID describe exactly what the option does.
regular/small/full should have `so_container_width` if we need to add an ID one day, no?

Up to ID suggestion otherwise

Reply:
Yep, fine. My concern was mainly about commit order here but did not check.